### ISSUE TITLE
Can't use inline HTML

### ISSUE THREAD

**Issue by antoinevg** [sbstjn/node-blog#3]
(Love your design philosophy, this looks like a highly promising tool, thank you for putting it out there!)

Trying to use inline HTML in a post as per Gruber's example escapes the tags and presents them as text.

_To reproduce:_ Create a post with the following text

```
This is a regular paragraph.

<table>
    <tr>
        <td>Foo</td>
    </tr>
</table>

This is another regular paragraph.
```

_Expected output:_

A table element with the text foo

Actual output:

The HTML code

**Comment by sbstjn** [comment]
Thanks for your awesome bug report :) I found out it's related to a bug in markdown-js available at evilstreak/markdown-js#34. The bug in markdown-js exists for about 11 months and a quick and dirty fix is attached in the linked issue as well. 

You can patch `node_modules/node-markdownblog/node_modules/markdown/lib/markdown.js` to get inline HTML working…

**Update:** With using the provided patch you will break HTML in `code` blocks :(

Maybe I should switch to another markdown compiler…

**Comment by antoinevg** [comment]
Fortunately I'm not using code blocks on this particular blog so it certainly sorts me out for the meantime, thank you muchly! :-)
